File: SDKs/zuora.com/2021-08-20/python/src/sdk/models/shared/customobjectbulkjoberrorresponse.py
import dataclasses
from typing import Any,Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class CustomObjectBulkJobErrorResponse:
    code: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('code') }})
    message: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('message') }})
    record: Optional[dict[str, Any]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('record') }})
    row: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('row') }})
    

File: SDKs/zuora.com/2021-08-20/typescript/src/sdk/models/shared/customobjectbulkjoberrorresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class CustomObjectBulkJobErrorResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=code" })
  code?: number;

  @SpeakeasyMetadata({ data: "json, name=message" })
  message?: string;

  @SpeakeasyMetadata({ data: "json, name=record" })
  record?: Record<string, any>;

  @SpeakeasyMetadata({ data: "json, name=row" })
  row?: number;
}
